airflow/providers/microsoft/azure/provider.yaml

```diff
@@ -64,7 +64,7 @@ dependencies:
   - azure-storage-common>=2.1.0
   - azure-storage-file>=2.1.0
   # Limited due to https://github.com/Azure/azure-uamqp-python/issues/191
-  - azure-servicebus>=7.6.1; platform_machine != "aarch64"
+  - azure-servicebus>=7.6.1
```

Comment:
Should we also remove or change comment?

Reply:
🤦 of course

Reply:
Fixed.